relational_gui.py

```diff
@@ -83,7 +83,7 @@ def printhelp(code=0):
         import signal
         signal.signal(signal.SIGINT, signal.SIG_DFL)
 
-        import sip  # needed on windows
+        #import sip  # needed on windows
```

Comment:
eh, but if I merge this PR it will no longer work on windows.

Do you think you can put some conditionals around the imports so the behaviour only changes on osx?